plugin.py

```diff
@@ -442,6 +442,18 @@ def branches(self, irc, msg, args, package):
         return
     branches = wrap(branches, ['text'])
 
+    def wiki(self, irc, msg, args, page_name):
+        """<wiki_page>
+
+        Return the Fedora wiki link for the specified page."""
+        link = "https://fedoraproject.org/wiki/{}".format(page_name)
+
+        # Properly format spaces for the wiki link.
+        link.replace(" ", "_")
+        irc.reply(link)
+        return
```

Comment:
You can remove the return line, since it's the last line of the function.
The other functions also don't feature it.